# HG changeset patch
# User andrew
# Date 1378870967 -3600
#      Wed Sep 11 04:42:47 2013 +0100
# Node ID 09f439dfd7b9d6e022f7af71c328b5b4b6a36c23
# Parent  24b23eb187fcfb801eb3d479b30e3321f9f5dfd6
Fix Zero following changes to entry_frame_call_wrapper in 8016131

diff --git a/src/cpu/zero/vm/entryFrame_zero.hpp b/src/cpu/zero/vm/entryFrame_zero.hpp
--- openjdk/hotspot/src/cpu/zero/vm/entryFrame_zero.hpp
+++ openjdk/hotspot/src/cpu/zero/vm/entryFrame_zero.hpp
@@ -58,8 +58,8 @@
                            JavaCallWrapper* call_wrapper,
                            TRAPS);
  public:
-  JavaCallWrapper *call_wrapper() const {
-    return (JavaCallWrapper *) value_of_word(call_wrapper_off);
+  JavaCallWrapper **call_wrapper() const {
+    return (JavaCallWrapper **) addr_of_word(call_wrapper_off);
   }
 
  public:
diff --git a/src/cpu/zero/vm/frame_zero.inline.hpp b/src/cpu/zero/vm/frame_zero.inline.hpp
--- openjdk/hotspot/src/cpu/zero/vm/frame_zero.inline.hpp
+++ openjdk/hotspot/src/cpu/zero/vm/frame_zero.inline.hpp
@@ -132,7 +132,7 @@
   return fp();
 }
 
-inline JavaCallWrapper* frame::entry_frame_call_wrapper() const {
+inline JavaCallWrapper** frame::entry_frame_call_wrapper_addr() const {
   return zero_entryframe()->call_wrapper();
 }
 
